doc/tutorials/02-charged_system/CMakeLists.txt

```diff
@@ -14,11 +14,7 @@ configure_tutorial_target(
   scripts/nacl_units.py
   scripts/nacl_units_vis.py)
 
-nb_export(
-  TARGET tutorial_02 SUFFIX "1" FILE "02-charged_system-1.ipynb" HTML_RUN
-  VAR_SUBST
-  "num_steps_equilibration=200;num_configs=100;integ_steps_per_config=250")
-nb_export(
-  TARGET tutorial_02 SUFFIX "2" FILE "02-charged_system-2.ipynb" HTML_RUN
-  VAR_SUBST
-  "num_steps_equilibration=200;num_configs=50;integ_steps_per_config=100")
+nb_export(TARGET tutorial_02 SUFFIX "1" FILE "02-charged_system-1.ipynb"
+          HTML_RUN)
+nb_export(TARGET tutorial_02 SUFFIX "2" FILE "02-charged_system-2.ipynb"
+          HTML_RUN)
```

Comment:
Tutorial 02 Part 2 was not touched by the charged rod PR. Without these variable substitutions, it takes 10 min to generate the HTML file. I hope this is not going to time out in CI in 1 week.